content/sensu-go/5.16/guides/monitor-sensu.md

```diff
@@ -0,0 +1,150 @@
+menu: 
+  sensu-go-5.16:
+    parent: guides
+---
+
+- [Monitor your Sensu backend instances](#monitor-your-sensu-backend-instances)
+- [Monitor your Sensu agent instances](#monitor-your-sensu-agent-instances)
+- [Monitor your Sensu API instances](#monitor-your-sensu-api-instances)
+- [Monitor your Sensu dashboard instances](#monitor-your-sensu-dashboard-instances)
+- [External etcd PLACEHOLDER](#external-etcd-placeholder)
+- [PostgreSQL PLACEHOLDER](#postgresql-placeholder)
+
+This guide describes best practices and strategies for monitoring Sensu with Sensu. 
+It explains how to ensure your Sensu components are properly monitored, including your Sensu backend, agent, API, and dashboard.
+
+To completely monitor Sensu (a Sensu backend with internal etcd and an agent), you will need at least one other independent Sensu instance.
+
+This guide requires Sensu plugins.
+For more information about using Sensu plugins, see [Install plugins with assets][10].
+
+_**NOTE**: This guide assumes you are not using Sensu clustering, although the strategies in this guide are still useful for monitoring cluster members._
+
+## Monitor your Sensu backend instances
+
+Monitor the host running the `sensu-backend` in two ways:
+
+* Locally by a `sensu-agent` process for operating system checks and metrics and Sensu services that are not part of the Sensu event system (like the dashboard)
+* Remotely from an independent Sensu instance for Sensu components that must be running for Sensu to create events.
+
+Here's an overview of the checks you might use depending on your backend setup:
+
+|  | Check type | Recommended plugin |
+---|------------|--------------------|
+Single backend | Service checks | Plugin name |
+Clustered backends | Port checks | Plugin name |
+Distributed single backend instances | Prometheus collector check | Plugin name |
+Distributed clustered backend instances | API queries as checks | Plugin name |
+Distributed single backend instances (federated) | Service checks, port checks | Plugin name |
+Distributed clustered backend instances (federated) | Prometheus collector check, API queries as checks | Plugin name |
+
+### Monitor the Sensu backend locally
+
+Monitor the host that the `sensu-backend` process runs on like any other node in your infrastructure.
+This includes checks and metrics for [CPU][1], [memory][2], [disk][3], and [networking][4].
+Find more plugins at [Bonsai, the Sensu asset index][5].
+
+### Monitor the Sensu backend remotely
+
+Monitor the `sensu-backend` server process from an independent Sensu instance.
+To do this, call Sensu's [health API endpoint][6] and use the [check-http plugin][7] with the following check definition:
+
+{{< highlight json >}}
+{
+  "type": "CheckConfig",
+  "api_version": "core/v2",
+  "metadata": {
+    "namespace": "default",
+    "name": "check_sensu_server_port"
+  },
+  "spec": {
+    "command": "check-http.rb -h remote-api-hostname -P 8080 -p /health --response-code 204",
```

Comment:
Response code expected should be a 200 instead of a 204.

Reply:
Also add the `-w` flag to print the output of the check, which would give a user more context on the actual failure.